dockerfiles/mlrun-kfp/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG MLRUN_VERSION=kfp
+FROM mlrun/mlrun:${MLRUN_VERSION}
```

Comment:
mlrun/mlrun comes from dockerhub and maybe during buildtime, you will build the images for other registries so the built image would not be `mlrun/mlrun` but maybe `x.com/mlrun/mlrun`. I suggest passing the registry image here too